docs/errors/E086.md

```diff
@@ -0,0 +1,12 @@
+# E086: redundant delete statement
```

Comment:
The error should explain *why* the `delete` is redundant.

* Is there two consecutive `delete` statements?
* Does the `delete` statement have no effect because the object is frozen?
* Does the `delete` statement have no effect because the operand is a variable?
* Does the `delete` statement have no effect because the operand is otherwise not a property lookup?